### ISSUE TITLE
Search-function is case-sensitive 

### ISSUE THREAD

**Issue by stefans73** [nextcloud/contacts#815]
Hello,

Bug:
Using the nextcloud search-function in contacts filters case sensitive. If I search for "stefan", then "Stefan" will not be found.

**To Reproduce**
1. Go to contacts
2. Click on search (top/right)
3. Type a name, once lowercase, once uppercase
4. You will find only names, written in the same case as written to the database.

**Expected behavior**
I would expect to find "Stefan" even when I search for "stefan"

**Actual behavior**
"Stefan" is not found, when searched for "stefan"

### Server configuration
**Operating system**: Linux

**Web server:** Apache

**Database:** MariaDB

**PHP version:** 7.0

**Nextcloud version:** 15.0.0

**Contacts version:** 3.0.1

**Updated from an older Nextcloud

**Signing status:**
```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
No errors have been found.

**List of activated apps:**
```
If you have access to your command line run e.g.:
Enabled:
  - accessibility: 1.1.0
  - activity: 2.8.2
  - auto_mail_accounts: 0.1.3
  - calendar: 1.6.4
  - cloud_federation_api: 0.1.0
  - comments: 1.5.0
  - contacts: 3.0.1
  - dav: 1.8.0
  - event_update_notification: 0.3.1
  - federatedfilesharing: 1.5.0
  - federation: 1.5.0
  - files: 1.10.0
  - files_antivirus: 2.0.0
  - files_pdfviewer: 1.4.0
  - files_sharing: 1.7.0
  - files_texteditor: 2.7.0
  - files_trashbin: 1.5.0
  - files_versions: 1.8.0
  - files_videoplayer: 1.4.0
  - firstrunwizard: 2.4.0
  - gallery: 18.2.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.3.0
  - mail: 0.11.0
  - nextcloud_announcements: 1.4.0
  - notes: 2.5.1
  - notifications: 2.3.0
  - oauth2: 1.3.0
  - password_policy: 1.5.0
  - provisioning_api: 1.5.0
  - serverinfo: 1.5.0
  - sharebymail: 1.5.0
  - support: 1.0.0
  - survey_client: 1.3.0
  - systemtags: 1.5.0
  - tasks: 0.9.8
  - theming: 1.6.0
  - twofactor_backupcodes: 1.4.1
  - updatenotification: 1.5.0
  - workflowengine: 1.5.0
Disabled:
  - admin_audit
  - encryption
  - files_external
  - user_external
  - user_ldap

**Nextcloud configuration:**
```
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "abc.de",
            "nc.abc.de"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "https:\/\/nc.abc.de\/",
        "check_for_working_wellknown_setup": true,
        "check_for_working_htaccess": true,
        "htaccess.RewriteBase": "\/",
        "dbtype": "mysql",
        "version": "15.0.0.10",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbtableprefix": "nc123_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true,
        "maintenance": false,
        "loglevel": 0,
        "logfile": "\/var\/www\/nextcloud_extern\/log\/nextcloud.log",
        "log_type": "owncloud",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "theme": "",
        "defaultapp": "apporder",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 6379
        },
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "25",
        "updater.release.channel": "stable",
        "auto_mail_accounts": {
            "imap_host": "localhost",
            "imap_port": "993",
            "imap_ssl_mode": "none",
            "smtp_host": "localhost",
            "smtp_port": "465",
            "smtp_ssl_mode": "none",
            "email_address_suffix": "@abc.de"
        }
    }
}

### Client configuration
**Browser:**
Firefox 60.4
**Operating system:**
Linux

**Comment by brettferrell** [comment]
Oh wow, this is totally the problem I was going to report, and this has given me the workaround.  I thought Searching was totally broken, but it is just that it is case sensitive!  Please fix this asap.

**Comment by skjnldsv** [comment]
@brettferrell this is fixed in 3.0.2. The app will be released next week :)
Thanks for using the search function and not creating your own issue ;)
